regolith/tools.py

```diff
@@ -2104,39 +2105,47 @@ def google_cal_auth_flow():
         token.write(creds.to_json())
     # Save the credentials for the next run
 
-def add_to_gitlab(name):
-    """Takes a newly created event, and adds it to the user's google calendar
+def create_talk_repo(name, rc):
+    """Takes in the presentation name and creates a repo
 
     Parameters:
         name - the name/key of the event
```

Comment:
I think strictly you can pass any string here and it will be used, so I would write this maybe like this:
```
    name - string
      the name that will be used for the repository. e.g., the key of presentation/projectum